src/hooks/useOrders.ts

```diff
@@ -69,7 +69,8 @@ export function useOrders(): Result {
           if (offset === 0) {
             // fresh start/refresh: replace whatever is stored
             setOrders(filteredOrders)
-          } else {
+            // only if we have something to add
+          } else if (filterDeletedOrders.length > 0) {
```

Comment:
I had that before on my super not complicated at all previous version :D

Reply:
And you were right to have it :+1: 
I somehow deleted it during refactor